### PR TITLE
Issue 5: Image Tags

### PR DESCRIPTION
**Pull request by twebner**

### CODE REVIEW

**Comment by twebner** [comment]
[Issue 5: Image Tags](https://trello.com/c/6LIHnzKS/405-issue-5-image-tags)

File: db/migrate/20210311224210_acts_as_taggable_on_migration.acts_as_taggable_on_engine.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This migration comes from acts_as_taggable_on_engine (originally 1)
```
Comment (cpanlasigui) [inline db/migrate/20210311224210_acts_as_taggable_on_migration.acts_as_taggable_on_engine.rb:1]: I think we will need to add a rule to exclude the acts_as_taggle_on migrations from rubobop

File: Gemfile [hunk 2]
```diff
@@ -23,8 +23,11 @@ gem 'simple_form'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'acts-as-taggable-on'
   gem 'byebug', platform: :mri
   # Use sqlite3 as the database for Active Record
+  gem 'pry'
+  gem 'pry-remote'
```
Comment (pauldanielm) [inline Gemfile:30]: Is pry used anywhere? If not I think we can probably remove it.

**Review by pauldanielm — COMMENTED** [review]
Changes made to fix rubocop issues can be squashed into their respective original commits.

File: Gemfile (hunk 2 above)
Comment (twebner) [reply Gemfile:30]: We've been using it for debugging and just decided to leave it in

**Review by cpanlasigui — APPROVED** [review]

**Pull request merged**